Example/Pods/Local Podspecs/Heyou.podspec.json

```diff
@@ -14,10 +14,11 @@
   },
   "source": {
     "git": "https://github.com/etoledom/Heyou.git",
-    "tag": "0.2.3"
+    "tag": "0.3.0"
   },
   "platforms": {
     "ios": "10.0"
   },
-  "source_files": "Heyou/Classes/**/*"
+  "source_files": "Heyou/Classes/**/*",
+  "swift_version": "4.2"
```

Comment:
We probably want this one to be `"swift_version": "4.2"`.
It fixes for me locally after running `pod install`, could you please try it again locally too and let me know if you have some diff here?

Reply:
Hi @etoledom  , I set **5.0** as default to not have to manually change it, using something like 
```
post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
       if ['Heyou'].include? target.name
          config.build_settings['SWIFT_VERSION'] = '5.0'
        end
    end
  end
end 
```
----------------------
Not sure why you see changes on `  "source_files": "Heyou/Classes/**/*"`

![Screen Shot 2019-12-17 at 14 57 30](https://user-images.githubusercontent.com/5015006/71021634-8d609600-20dd-11ea-9ab0-716130a1689b.png)